Add rem preferences (#54)
zeplin/zeplin-extension-documentation: model/remPreferences.md

## RemPreferences
**Kind**: global interface

<a name="RemPreferences"></a>
* [RemPreferences](#RemPreferences)
    * [.useForFontSizes](#RemPreferences+useForFontSizes) : <code>Boolean</code>
    * [.useForMeasurements](#RemPreferences+useForMeasurements) : <code>Boolean</code>
    * [.rootFontSize](#RemPreferences+rootFontSize) : <code>Number</code>

<a name="RemPreferences+useForFontSizes"></a>
### remPreferences.useForFontSizes : <code>Boolean</code>

> Indicates whether the rem is enabled for font sizes or not.

Reviewer: ```suggestion
Whether rem unit is used for font sizes.
```

**Kind**: instance property of [<code>RemPreferences</code>](#RemPreferences)

<a name="RemPreferences+useForMeasurements"></a>
### remPreferences.useForMeasurements : <code>Boolean</code>

> Indicates whether the rem is enabled for measurement (e.g. width and height) or not.

Reviewer: ```suggestion
Whether rem unit is used for measurements (e.g. width, height, etc.).
```

**Kind**: instance property of [<code>RemPreferences</code>](#RemPreferences)

<a name="RemPreferences+rootFontSize"></a>
### remPreferences.rootFontSize : <code>Number</code>

> The value of root font size

Reviewer: ```suggestion
The value of the root element's font size.
```
